nuget.props

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
+<Project>
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <Authors>Phillip Trelford, Ruben Bartelink, Milos Chaloupka</Authors>
+    <Copyright>Copyright © 2010-21 Phillip Trelford, Ruben Bartelink, Milos Chaloupka</Copyright>
+    <PackageProjectUrl>https://github.com/fsprojects/TickSpec</PackageProjectUrl>
+    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
```

Comment:
also add packagelincense like this https://github.com/jet/propulsion/blob/master/Directory.Build.props#L8

Reply:
Are you sure that the way you are proposing works? According to [this](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets), the following two values are supported `PackageLicenseFile` and `PackageLicenseExpression`. We can change it from file to expression though.

Reply:
the expression flows to the packed nupkg and helps with some things.
there is awarning that dotnet pack gives if you dont do it I think